packages/augur-ui/src/services/browser-mesh.ts

```diff
@@ -95,17 +96,19 @@ export async function createBrowserMesh(
     throw new Error(`Attempting to create browser mesh without it being enabled in config ${JSON.stringify(config)}`);
   }
 
-  const meshConfig = createBrowserMeshConfig(
-    config.ethereum.http,
-    web3Provider,
-    Number(config.networkId),
-    config.zeroX.mesh.verbosity || 5,
-    config.zeroX.mesh.useBootstrapList,
-    config.zeroX.mesh.bootstrapList
-  );
+  if (!isGoogleBot()) {
```

Comment:
I think you should just put this check up one level where we actually create/read the `config` and just force `config.mesh.enabled = false`, then ALL of the places that check for mesh will be covered.

So, e.g. somewhere around here:
https://github.com/AugurProject/augur/blob/master/packages/augur-ui/src/modules/app/actions/init-augur.ts#L219

Reply:
Makes way more sense 👍